service/src/main/java/gov/nasa/pds/api/engineering/elasticsearch/business/RequestAndResponseContext.java

```diff
@@ -161,10 +161,17 @@ private RequestAndResponseContext(
 		}
 		else
 		{
+			String known = "";
 			log.warn("Could not find a matach for application type: " + String.valueOf(this.format));
 			log.warn("   Known types: " + String.valueOf(this.formatters.keySet().size()));
-			for (String key : this.formatters.keySet()) log.warn("      key: " + String.valueOf(key));
-			throw new ApplicationTypeException("The given application type, " + String.valueOf(this.format) + ", is not known by RquestAndResponseContext.");
+			for (String key : this.formatters.keySet())
```

Comment:
Maybe you could use a method like 

String known = String.join(",", this.formatters.keySet());

That would be shorter and I feel like that would be more understandable as well but I did not test it.

Reply:
resolved